schedule/functional/extra_tests_clamav.yaml

```diff
@@ -0,0 +1,11 @@
+name: extra_tests_clamav
+description:    >
+    Maintainer: QE Core.
+    Extract 'clamav' module from 'extra_tests_textmode' to meet the RAM
+    requirements.
+schedule:
+    - installation/bootloader_start
+    - boot/boot_to_desktop
+    - console/consoletest_setup
+    - console/clamav
+    - console/coredump_collect
```

Comment:
Just curious to know why `coredump_collect` test is also included?

Reply:
I thought it was necessary in the end of all testsuites :sweat_smile:

Reply:
I think that you should keep the coredump_collect test, here :) It can probably give some coredumps that are not visible during the test.

Reply:
On the other hand, I don't see the need of `consoletest_setup`, especially if it has already run in a previous test.

Reply:
Seems legit :)
https://openqa.suse.de/tests/8393452